phpBB/includes/functions_posting.php

```diff
@@ -2490,7 +2490,7 @@ function submit_post($mode, $subject, $username, $topic_type, &$poll_ary, &$data
 *				- 'topic_last_poster_name'
 *				- 'topic_last_poster_colour'
 * @param int $bump_time The time at which topic was bumped, usually it is a current time as obtained via time().
-* @return string An URL to the bumped topic, example: ./viewtopic.php?forum_id=1&amptopic_id=2&ampp=3#p3
+* @return string An URL to the bumped topic, example: ./viewtopic.php?amptopic_id=2&ampp=3#p3
```

Comment:
./viewtopic.php?topic_id=2&amp;p=3#p3

Reply:
Actually, to be consistent with the new code, it should be:
./viewtopic.php?p=3#p3